quora-service/src/main/java/com/upgrad/quora/service/business/AnswerBusinessService.java

```diff
@@ -48,4 +50,41 @@ public AnswerEntity createAnswer(final AnswerEntity answerEntity, final String q
         answerEntity.setQuestion(questionEntity);
         return answerDao.createAnswer(answerEntity);
     }
+
```

Comment:
Add this line above deleteAnswer method:

```suggestion
@Transactional(propagation = Propagation.REQUIRED)
```